providers-sdk/v1/upstream/gql/vulnmgmt_gql.go

```diff
@@ -0,0 +1,208 @@
+package gql
```

Comment:
The GQL queries are in core cnquery, because I assume we need to use them in different providers. Currently it is only os, but vsphere will follow and perhaps more.